docs/wiki/EL/el-specs.md

```diff
@@ -961,6 +961,76 @@ TODO
 
 TODO
 
+### Gas Accounting Examples
+Up to this point, we've talked about EL post-merge gas mechanics in a variety of scenarios. Let's tie it all together with some examples.
+
+Note: Gas parameters will differ depending upon if the contract was created before [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) was implemented.
```

Comment:
I believe this should be based on tx type - whether user wallet is using 1559 or not.  

What could be clearer here is that we have multiple types of transactions - legacy, 1559 and blob carrying

Reply:
Exactly, see this for hint https://github.com/ethereum/execution-spec-tests/blob/328b7f49f56e0a1b05d915f4e6a918e8391d22db/src/ethereum_test_types/types.py#L761-L794

Reply:
hey @taxmeifyoucan @raxhvl, great point that the gas params differ based on tx type. I updated the PR to account for this in latest commit https://github.com/eth-protocol-fellows/protocol-studies/pull/441/commits/6656bb318ad46196b5a493d54ce19fea9c0033e0.  

Additionally, I made the comparisons in the PR to be more tabular instead of bullet points. Def more readable